.github/infrastructure/docker-compose-kafka.yaml

```diff
@@ -0,0 +1,21 @@
+version: '3.9'
+services:
+  component:
+    container_name: dapr-kafka-pluggable-component
+    image: "ghcr.io/mcandeia/kafka-pluggable-component:latest"
```

Comment:
Can we have these images published to dapr-sandbox?

Reply:
Definitely yes